test/test_finance_utils.py

```diff
@@ -40,19 +40,19 @@ def test_5():
 
 
 @pytest.mark.webtest
-def test_6():
+def test_update_all_symbols():
     d = 'stock_db/empty2'
     start_date = datetime.date(2000, 1, 1)
     end_date = datetime.date.today()
     fu.update_all_symbols(d, start_date, end_date)
     assert len(fu.get_all_symbols(d)) == 1
 
 
-def test_7():
+def test_get_symbols_from_file():
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'test_get_symbols_from_file' from line 7 (F811)](https://app.codacy.com/manual/mathieugouin/tradesim/pullRequest?prid=5529150)